src/precice/bindings/c/SolverInterfaceC.h

```diff
@@ -242,6 +242,8 @@ void precicec_mapWriteDataFrom ( int fromMeshID );
  */
 void precicec_mapReadDataTo ( int toMeshID );
 
+const char* precicec_actionWriteIterationCheckpoint();
+const char* precicec_actionReadIterationCheckpoint();
 
```

Comment:
It looks like the following functions are missing in the C interface:
```
const std::string& actionWriteInitialData();
const std::string& actionPlotOutput();
int exportVTK();
int exportAll();
```

Reply:
If we ignore

```
const std::string& actionPlotOutput();
int exportVTK();
int exportAll();
```

only 

```
const std::string& actionWriteInitialData();
```

is missing.

Reply:
@BenjaminRueth promised that he would add `actionWriteInitialData` in another PR :smirk: